mm2src/mm2_main/src/lp_swap/recreate_swap_data.rs

```diff
@@ -73,15 +73,18 @@ pub async fn recreate_swap_data(ctx: MmArc, args: RecreateSwapRequest) -> Recrea
         },
         InputSwap::SavedSwap(SavedSwap::Taker(taker_swap)) | InputSwap::TakerSavedSwap(taker_swap) => {
             recreate_maker_swap(ctx, taker_swap)
+                .await
                 .map(SavedSwap::from)
                 .map(|swap| RecreateSwapResponse { swap })
         },
     }
 }
 
-fn recreate_maker_swap(ctx: MmArc, taker_swap: TakerSavedSwap) -> RecreateSwapResult<MakerSavedSwap> {
+async fn recreate_maker_swap(ctx: MmArc, taker_swap: TakerSavedSwap) -> RecreateSwapResult<MakerSavedSwap> {
```

Comment:
What is the reason of this change? I can't see this function involves with async calls inside.

Reply:
i added lp_coinfind call at the end, which prompted this await. it's only active in the new target only tho (for now).